Earthfile

```diff
@@ -0,0 +1,21 @@
+VERSION 0.8
+
+nanoc:
+	#FROM docker.io/library/ruby:3.3-alpine
+	FROM docker.io/library/ruby:3.3
```

Comment:
Is there a reason to not use alpine here?

Reply:
There was a Gemfile.lock conflict blocking successful FFI installation due to differences between linux-amd64-gnu and linux-amd64-musl and I neither wanted to break compatibility with running directly nor perform a deeper investigation.